elec/layout/default/servo-drive.kicad_pro

```diff
@@ -277,7 +278,7 @@
       "gencad": "",
       "idf": "",
       "netlist": "../../../build/default.net",
-      "plot": "",
+      "plot": "/Users/napowderly/Projects/atoFab/",
```

Comment:
Can/should this be a relative path?

Reply:
Should be! Great pick

Seems like this something KiCAD would've just added